fsd_config/form_jsons/cof_r3/en/local-support-cof-r3-w1.json

```diff
@@ -43,10 +44,10 @@
               "acceptedFiles": "image/jpeg,image/png,application/pdf,text/plain,application/msword,application/vnd.openxmlformats-officedocument.wordprocessingml.document,application/vnd.oasis.opendocument.text,text/csv,application/vnd.ms-excel,application/vnd.openxmlformats-officedocument.spreadsheetml.sheet,application/vnd.oasis.opendocument.spreadsheet"
             },
             "showNoScriptWarning": false,
-            "minimumRequiredFiles": 1
+            "minimumRequiredFiles": 0
           },
           "type": "ClientSideFileUploadField",
-          "title": "Upload supporting evidence (optional)",
```

Comment:
@adamdavies1 The optional text should still be included in the title:
https://fsd-pre-award.herokuapp.com/cof-r3/application-pages/strategy/strategy-05

Reply:
It does, basically when we change it to 0 required files, it will add the optional tag
![image](https://github.com/communitiesuk/digital-form-builder/assets/97108643/2eefbd2b-8c82-464a-b3e8-7ff49abef7af)